src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+
+    public RightTriangle(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side * Math.sqrt(3.0) / 4;
+    }

```

Comment:
it looks strange, you can calculate it by two legs
```suggestion
    public double getArea() {
        return side * side * Math.sqrt(3.0) / 4;
    }

```